transformer_engine/pytorch/csrc/extensions/dropout.cpp

```diff
@@ -0,0 +1,133 @@
+/*************************************************************************
+ * Copyright (c) 2022-2025, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ *
+ * See LICENSE for license information.
+ ************************************************************************/
+
+#include "transformer_engine/dropout.h"
+
+#include <ATen/cuda/CUDAGeneratorImpl.h>
+
+#include <ATen/cuda/CUDAGraphsUtils.cuh>
+
+#include "common.h"
+#include "pybind.h"
+#include "transformer_engine/transformer_engine.h"
+
+namespace transformer_engine::pytorch {
+
+void unpack(at::PhiloxCudaState arg, int64_t *rng_state_ptr) {
```

Comment:
We already have exactly the same function in attention.cpp, we should just reuse it.

Reply:
@timmoon10 can you advise on how to move this function to common.h and reuse it in attention.cpp and dropout.cpp?

Reply:
`unpack` is simple enough that I think it's simpler to just include it within `dropout_fwd`.